pilot/pkg/networking/core/v1alpha3/httproute_test.go

```diff
@@ -195,6 +196,98 @@ func TestGenerateVirtualHostDomains(t *testing.T) {
 	})
 }
 
+func TestSidecarOutboundHTTPRouteConfigWithVirtualServiceHosts(t *testing.T) {
```

Comment:
donot we have case ` "sidecar config with allow any and virtual service includes non existing service"` for this in original L858

Reply:
But this is which subsets to validate destination clusters are generated correctly